.github/workflows/daily-runtime-validation.yml

```diff
@@ -53,6 +59,7 @@ jobs:
           python3 test-runner/wasi_test_runner.py \
             -r ./adapters/${{ matrix.runtime }}.py \
             --json-output-location results.json \
+            -f ./adapters/${{ matrix.runtime }}_skip.json \
```

Comment:
I wonder if that's something we can set conditionally instead of creating empty `skip` files for other adapters (e.g. set `EXTRA_RUNNER_PARAMS` with default to be empty)?

Also, my preference would be to actually run all the tests in the daily runtime validation - it's fine if that fails, it is only used for reporting, so I'd rather keep all of them running.

Reply:
got it, will change in a bit

Reply:
done. https://github.com/WebAssembly/wasi-testsuite/pull/55/commits/20bc89e67e7a253ac0b18ec3c3838444da4e959e

Reply:
> Also, my preference would be to actually run all the tests in the daily runtime validation - it's fine if that fails, it is only used for reporting, so I'd rather keep all of them running.

i tend to disagree. always failing tests will be ignored by people. it's better to have a skip list and/or expected-fail list.

Reply:
The daily job is to generate a list of runtimes and their wasi readiness. The results are not being used currently at all, but once we update all the tests so they're 1:1 with WASI spec we'll start publishing report to a website. 
Runtimes, if they choose to use wasi testsuite in the CI, can (and should, for expected failures until they're fixed) use exclusion list.

Reply:
what do you mean by 1:1?

> we'll start publishing report to a website.

i feel it's simpler/better to let each runtimes publish their results by themselves if they want.

Reply:
we added code to add the dot, dot-dot at the cost of an extra stat for all users. We no longer require the skip

Reply:
actually, it is more expensive than this, as all files end up needing inodes. I'll add a CLI arg to wazero as it is very expensive to comply with this and few will need it.